ims/ostis_tech/unificated_models/section_unificated_models_kpm/section_subject_domain_of_situations_and_events/content_html/introduction_for_subject_domain_of_situations_and_events_in_sc_memory.html

```diff
@@ -4,7 +4,7 @@
 	<li>к логическому удалению актуальных <sc_element sys_idtf = "sc_element">sc-элементов</sc_element>, т.е. к переводу их в неактуальное состояние (это необходимо для хранения протокола изменения состояния базы знаний, в рамках которого могут описываться действия по удалению <sc_element sys_idtf = "sc_element">sc-элементов</sc_element>);</li>
 	<li>к возврату логически удаленных <sc_element sys_idtf = "sc_element">sc-элементов</sc_element> в статус актуальных (необходимость в этом может возникнуть при откате базы знаний к какой-нибудь ее прошлой версии);</li>
 	<li>к физическому удалению <sc_element sys_idtf = "sc_element">sc-элементов</sc_element>;</li>
-	<li>к изменению состояния актуальных (логически не удаленных <sc_element sys_idtf = "sc_element">sc-элементов</sc_element>) – <sc_element sys_idtf = "sc_node">sc-узел</sc_element> может превратиться в <sc_element sys_idtf = "sc_edge">sc-ребро</sc_element>, <sc_element sys_idtf = "sc_edge">sc-ребро</sc_element> может превратиться в <sc_element sys_idtf = "sc_arc">sc-дугу</sc_element>, <sc_element sys_idtf = "sc_arc">sc-дуга</sc_element> может поменять направленность, <sc_element sys_idtf = "sc_arc">sc-дуга</sc_element> общего вида может превратиться в <sc_element sys_idtf = "sc_arc_constant_permanent_membership">константную стационарную sc-дугу принадлежности</sc_element>, и т.д.;</li>
+	<li>к изменению состояния актуальных (логически не удаленных <sc_element sys_idtf = "sc_element">sc-элементов</sc_element>) – <sc_element sys_idtf = "sc_node">sc-узел</sc_element> может превратиться в <sc_element sys_idtf = "sc_common_edge">sc-ребро</sc_element>, <sc_element sys_idtf = "sc_common_edge">sc-ребро</sc_element> может превратиться в <sc_element sys_idtf = "sc_arc">sc-дугу</sc_element>, <sc_element sys_idtf = "sc_arc">sc-дуга</sc_element> может поменять направленность, <sc_element sys_idtf = "sc_arc">sc-дуга</sc_element> общего вида может превратиться в <sc_element sys_idtf = "sc_arc_constant_permanent_membership">константную стационарную sc-дугу принадлежности</sc_element>, и т.д.;</li>

```

Comment:
how is this possible? sc-node becoming sc-edge?

Reply:
I don't know. I don't think that's true now. I hadn't seen it in the Standard